src/data/uma-nickname-requirements.csv

```diff
@@ -105,11 +105,11 @@
 "長距離GⅠと短距離GⅠをそれぞれ勝利する","Win a GI race of short and long distance each once"
 "ファン数が500000人以上になる","Earn a total of ≥500000 fans"
 "GⅠを10勝する","Win a GI race 10 times"
-"全てのトレーニングレベルが3以上で育成を完了する","Complete the training with all infrastructure at level ≥3"
-"短距離、マイル、中距離、長距離、ダートの重賞を\nそれぞれ1勝ずつする","Win a dirt big race of every distance each once"
+"全てのトレーニングレベルが3以上で育成を完了する","Complete the training with all trainings at level ≥3"
```

Comment:
trainings -> facilites?

Reply:
I don't like using words like infrastructure or facilities here since they're not in the original. The conflict we're having comes from translating 育成 as Training and the japanese also using トレーニング which is also training. Would prefer something like:
```suggestion
"全てのトレーニングレベルが3以上で育成を完了する","Complete a training run with all training levels at ≥3"
```

Reply:
I like the `training levels at`

Reply:
I'll commit it as such then.